### PR TITLE
Fix malformed stackdriver tracing env var

### PR DESCRIPTION
**Pull request by gargnupur**
Fix malformed stackdriver tracing env var
Ref: https://github.com/istio/istio/issues/20851

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[X ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by bianpengyuan — APPROVED** [review]

**Comment by gargnupur** [comment]
>   You appear to be fixing a bug in Go code, yet your PR doesn't include updates to any test files. Did you forget to add a test?
> 
> _Courtesy of your friendly test nag_.

Test coming in, different PR.. don't want to cause merge conflicts because of a new test in cherry-picks..

File: pkg/bootstrap/option/instances.go [hunk 1]
```diff
@@ -163,14 +163,17 @@ func StackDriverDebug(value bool) Instance {
 }
 
 func StackDriverMaxAnnotations(value int64) Instance {
+	//return newOption("stackdriverMaxAnnotations", value).withConvert(int64Converter(value))
```
Comment (howardjohn) [inline pkg/bootstrap/option/instances.go]: nit: remove

File: pkg/test/util/retry/retry.go [hunk 2]
```diff
@@ -80,6 +80,7 @@ func UntilSuccessOrFail(t test.Failer, fn func() error, options ...Option) {
 	t.Helper()
 	err := UntilSuccess(fn, options...)
 	if err != nil {
+		time.Sleep(time.Minute * 10)
```
Comment (howardjohn) [inline pkg/test/util/retry/retry.go]: remove this

File: tests/integration/telemetry/tracing/stackdriver/tracing_test.go [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+// More information on distributed tracing can be found here: https://istio.io/docs/tasks/telemetry/distributed-tracing/stackdriver/
+func TestProxyTracing(t *testing.T) {
+	framework.NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Run(func(ctx framework.TestContext) {
+			t.Logf("os STACKDRIVER_TRACING_MAX_NUMBER_OF_ATTRIBUTES %v", os.Getenv("STACKDRIVER_TRACING_MAX_NUMBER_OF_ATTRIBUTES"))
+			retry.UntilSuccessOrFail(t, func() error {
+				util.SendTraffic(ingInst, t, "Sending traffic", "", "", 1)
+
+				return fmt.Errorf("cannot get traces from stackdriver %v", bookinfoNsInst)
```
Comment (howardjohn) [inline tests/integration/telemetry/tracing/stackdriver/tracing_test.go]: how is this testing stack driver?

**Review by howardjohn — CHANGES_REQUESTED** [review]

File: pkg/bootstrap/option/instances.go (hunk 1 above)
Comment (gargnupur) [reply pkg/bootstrap/option/instances.go]: Those files were added by a bad commit.. they are removed now

File: pkg/test/util/retry/retry.go (hunk 2 above)
Comment (gargnupur) [reply pkg/test/util/retry/retry.go]: Those files were added by a bad commit.. they are removed now

File: tests/integration/telemetry/tracing/stackdriver/tracing_test.go (hunk 3 above)
Comment (gargnupur) [reply tests/integration/telemetry/tracing/stackdriver/tracing_test.go]: It had tracing enabled through stackdriver but it's not complete yet and I will add the test in a different PR.
Those files were added by a bad commit.. they are removed now

**Review by howardjohn — APPROVED** [review]

**Review by elfinhe — APPROVED** [review]

**Pull request merged**